test/bigtest/network/factories/groupFundFiscalYear.js

```diff
@@ -0,0 +1,9 @@
+import { Factory, faker } from '@bigtest/mirage';
+
+export default Factory.extend({
+  id: faker.random.uuid,
+  ledgerId: faker.random.uuid,
+  fundId: faker.random.uuid,
+  fiscalYearId: faker.random.uuid,
+  available: 1000,
```

Comment:
`faker.random.number({min:100, max:1000})` or `faker.finance.amount(1000,10000,2)` ?
Do you really need `ledgerId`, `fundId` and `fiscalYearId` here? IMO, we always need to redefine them in tests

Reply:
You are right